opat/drupal-9.yaml

```diff
@@ -746,5 +746,492 @@ chapters:
           - name: "web"
             adherence:
               level: "not-applicable"
+  # Note that this will likely be replaced with an automated process.
```

Comment:
@mgifford not sure what you meant by this comment since this file will be input.